library/psa_crypto.c

```diff
@@ -288,7 +288,8 @@ static uint8_t psa_get_drivers_initialized(void)
 int psa_can_do_hash(psa_algorithm_t hash_alg)
 {
     (void) hash_alg;
-    return psa_get_drivers_initialized();
+    /* Workaround for the legacy CryptoCell driver requiring hash during init */
+    return 1; //psa_get_drivers_initialized();
```

Comment:
Its a nit but the comment after the return statement is no there in the original patch and probably added by accident, right?

Reply:
It's actually there in the patch: https://github.com/zephyrproject-rtos/trusted-firmware-m/blob/main/lib/ext/mbedcrypto/0006-Enable-psa_can_do_hash.patch

Reply:
Oh, I was looking at the wrong revision it seems. Never mind then!